test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/KieServerWithWorkbenchSurvivalIntegrationTest.java

```diff
@@ -0,0 +1,160 @@
+        processServicesClient = KieServerClientProvider.getProcessClient(deploymentScenario.getKieServerDeployment());
+        queryServicesClient = KieServerClientProvider.getQueryClient(deploymentScenario.getKieServerDeployment());
+    }
+
+    @After
+    public void tearDown() {
+        InstanceLogUtil.writeDeploymentLogs(deploymentScenario.getWorkbenchDeployment());
+        InstanceLogUtil.writeDeploymentLogs(deploymentScenario.getKieServerDeployment());
+    }
+
+    @Test
+    public void kieServerScaleTest() {
+        KieServerInfo serverInfo = kieServicesClient.getServerInfo().getResult();
+        checkServerTemplateIsRegistred(serverInfo.getServerId(), serverInfo.getName());
+
+        scaleKieServerTo(0, serverInfo.getServerId());
+
+        scaleKieServerTo(1, serverInfo.getServerId());
+
+        logger.debug("Register Kie Container to Kie Server");
+        kieServerMgmtControllerClient.saveContainerSpec(serverInfo.getServerId(), serverInfo.getName(), CONTAINER_ID, CONTAINER_ALIAS, PROJECT_GROUP_ID, PROJECT_NAME, PROJECT_VERSION, KieContainerStatus.STARTED);
+        KieServerClientProvider.waitForContainerStart(deploymentScenario.getKieServerDeployment(), CONTAINER_ID);
+
+        logger.debug("Start process instance");
+        Long signalPid = processServicesClient.startProcess(CONTAINER_ID, SIGNALTASK_PROCESS_ID);
+        assertThat(signalPid).isNotNull().isGreaterThan(0L);
+        assertThat(queryServicesClient.findProcessInstances(0, 10)).isNotNull().hasSize(1);
+
+        scaleKieServerTo(0, serverInfo.getServerId());
+
+        logger.debug("Start process instance");
+
+        Assertions.assertThatThrownBy(() -> {
+            processServicesClient.startProcess(CONTAINER_ID, SIGNALTASK_PROCESS_ID);
+        }).isInstanceOf(KieServicesHttpException.class).hasMessageContaining("The host doesn't exist.");   
+//        try {
+//            Long noContainerPid = processServicesClient.startProcess(CONTAINER_ID, SIGNALTASK_PROCESS_ID);
+//            Assertions.fail("Container was registred on not existing server template with id " + noContainerPid);
+//        } catch (KieServerHttpRequestException ex) {
+//            // ok - requested URL does not exists
+//            System.out.println("KieServerHttpRequestException catched:\n" + ex.toString());
+//        } catch (KieServicesHttpException ex) {
+//            // ok - response code 503 Forbidden
+//            // The application is currently not serving requests at this endpoint. It may not have been started or is still starting.
+//            System.out.println("KieServicesHttpException catched:\n" + ex.toString());
+//        }
+
+        scaleKieServerTo(1, serverInfo.getServerId());
+
+        logger.debug("Wait for Kie Container start again");
+        KieServerClientProvider.waitForContainerStart(deploymentScenario.getKieServerDeployment(), CONTAINER_ID);
```

Comment:
No need to wait for container to start. When Kie server is scaled then it should contain all containers.